packages/acs-ui-common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internal/acs-ui-common",
-  "version": "1.7.0-beta.1",
+  "version": "1.7.0",
```

Comment:
Package versions should be `1.7.1-beta.0`, without the beta prefix we don't get the option `prerelease` in the changelog command